aws-gen.opam

```diff
@@ -20,7 +20,7 @@ depends: [
   "cmdliner" {>= "1.1.0"}
   "fmt"
   "yojson" {>= "1.6.0"}
-  "ocaml-migrate-parsetree" {< "2.0.0"}
+  "ocaml-compiler-libs"
```

Comment:
Can you add a lower bound on this? `>= "v0.11.0"` seems sensible via ppxlib.

I have some re-writing passes in the works that need ppxlib.